jumpscale/sals/reservation_chatflow/deployer.py

```diff
@@ -477,7 +477,17 @@ def show_payment(self, pool, bot):
         wallets = j.sals.reservation_chatflow.reservation_chatflow.list_wallets()
         wallet_names = []
         for w in wallets.keys():
-            wallet_names.append(w)
+            wallet = j.clients.stellar.get(w)
+            try:
+                balances = wallet.get_balance().balances
+            except:
+                continue
+            for balance in balances:
+                if balance.asset_code in escrow_asset:
```

Comment:
can join the two conditions here in one 

```suggestion
                if balance.asset_code in escrow_asset and balance.balance > total_amount :
```

Reply:
@xmonader  I did it separately to break if the balance isnt enough but keep iterating in the balances if the asset didnt match yet. if they are combined then we'll remove the break and it will iterate even if the balance wasn't matching. Should that matter?